scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/SchedulerAuthentication.java

```diff
@@ -97,14 +105,18 @@ public Scheduler login(Credentials cred) throws LoginException, AlreadyConnected
         String user = unPrincipal.getName();
 
         logger.info("user : " + user);
+
+        // get AWS S3 credentials
+        getAWSCredentials();
```

Comment:
This method gets the S3 or AWS credentials but I don't see a variable which receives those credentials. I would expect AwsCredentials awsCredentials = getAWSCredentials(); 

Maybe better name is like loadAwsCredentials...........()

Best would be if you could write a method which gets the credentials and then has the credentials and then does something with the credentials. So that it is clear for the reader what happens and where does it happen.